README.md

```diff
@@ -23,11 +23,16 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   
 ### Application can work at two modes:
 -  Local mode
-  -  All the ROS nodes run at the same machine.
+  -  All the functions run at the same machine.
 -  Network mode
-  -  ROS nodes run at two different machine: Server and Client.
-  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
-  -  Server runs all other ROS nodes except two nodes running on client.
+  -  Functions are distributed to two different machine: Server and Client.
+  -  Server Machine:
+    -  Runs perceptions nodes such as surface detection
+    -  Runs motion planning nodes such as the process path planner and free motion planner (moveit).
+  -  Client Machine:
+    -  Runs Rviz for user front-end and visualization only.
+    -  Runs either industrial_robot_simulator node or robot_driver node depending on whether it is on sim or real mode.
+    -  Runs either generate_point_cloud node or Kinect sensor driver node also based on the mode being used.
 
```

Comment:
If I'm not mistaken, this segment is the explanation I provided a few days ago.  I didn't do the best job at drafting this because I did not intent for this same exact description to be included in the README.  Even though, the right ideas are laid out here I think this can be polished a bit more so that it is clearer to users who aren't particularly familiar with this software.